src/math.ts

```diff
@@ -320,6 +320,8 @@ export const toSqrtPrice = (value: bigint, offset = 0n): bigint => {
   return value * 10n ** (SQRT_PRICE_SCALE - offset)
 }
 
+export const toPrice = toSqrtPrice
```

Comment:
toPrice function should return `Price` type

Reply:
I will make sure to adjust it when I am adding types to the TS, currently none of these have a type, they are just `bigint`